tests/util/blockdev_utils.rs

```diff
@@ -75,11 +78,24 @@ pub fn get_size(path: &Path) -> TestResult<Sectors> {
     };
 }
 
-fn wipe_header(path: &Path) -> TestResult<()> {
+pub fn wipe_header(path: &Path) -> TestResult<()> {
     let mut f = try!(OpenOptions::new().write(true).open(path));
     let zeroed = [0u8; SECTOR_SIZE * 16];
 
     try!(f.write_all(&zeroed[..SECTOR_SIZE * 16]));
 
     Ok(())
 }
+
+#[allow(dead_code)]
```

Comment:
Yes.  I'm not sure why - I've tried to figure it out a couple times but never resolved it.